boot/zephyr/boards/nrf54l15dk_nrf54l15_cpuapp.conf

```diff
@@ -0,0 +1,14 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+CONFIG_BOOT_MAX_IMG_SECTORS=512
```

Comment:
What is the sector size? Assuming 4KiB, this means you expect each slot to be 4*512 = 2MiB - this seems very wrong?